Docker/test/selenium/helpers/default-functions.js

```diff
@@ -196,4 +198,15 @@ const defaultFunctions = function () {
 
 };
 
-module.exports = defaultFunctions;
+module.exports = {
+	init: defaultFunctions,
+	skipIfExtensionNotPresent: function ( test, extension ) {
+		const result = browser.config.installed_extensions;
+
+		if ( result && result.includes( 'Wikibase' ) && result.includes( extension ) ) {
```

Comment:
This calls `String.includes`, right? Maybe we should `.split( ',' )` the string first?

(Also, “result” as a variable name doesn’t tell me much, to be honest.)